src/strategies/darkforest-score/index.ts

```diff
@@ -0,0 +1,42 @@
+import { multicall } from '../../utils';
+
+export const author = 'cha0sg0d';
+export const version = '0.1.0';
+
+const SCORE_INDEX = 5;
+const abi = [
+  'function players(address key) public view returns (bool, address, uint256, uint256, uint256, uint256)'
+];
+
+const calcScore = (score: number) => {
+  return score == 0 ? 0 : Math.floor(Math.log2(score))
+}
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const response = await multicall(
+    network,
+    provider,
+    abi,
+    addresses.map((address: any) => [
+      options.address,
+      'players',
+      [address.toLowerCase()]
+    ]),
```

Comment:
`players` method is taking too long to resolve and failing the tests when we pass more addresses, everything else looks good. make sure it works well and pass all tests when you run

```
npm test --strategy=darkforest-score --more=500
```

Reply:
Switched to subgraph for performance, and this passed all tests locally. Thanks!